examples/api-dotenv-request/app/Main.hs

```diff
@@ -0,0 +1,31 @@
+import Configuration.Dotenv (loadFile, defaultConfig)
+import System.Environment (lookupEnv)
+import Network.HTTP.Req
+import qualified Data.ByteString.Char8 as B
+import qualified Data.Text as T
+import Data.Aeson
+
+main :: IO ()
+main = do
+    loadFile defaultConfig
+    
+    --  Loading API URI from dotenv 
+    maybeApiEndpoint <- lookupEnv "API_URI"
+
+    case maybeApiEndpoint of
+        Just apiEndpoint -> do
+            -- Making the request
+            let apiEndpointURL = https (T.pack apiEndpoint) /: T.pack "posts" /: T.pack "1"
+            
+            --  Fake user agent
+            let userAgent = "Mozilla/5.0 (Linux; Android 5.0.1; Nokia 1100 LTE Build/GRK39F) AppleWebKit/536.14 (KHTML, like Gecko)  Chrome/55.0.3442.377 Mobile Safari/600.3"
+            let request :: Req (JsonResponse Value)
+                request = req GET apiEndpointURL NoReqBody jsonResponse $ header (B.pack "User-Agent") (B.pack userAgent)
+
+
+            -- Obtaining the response
+            response <- runReq defaultHttpConfig $ do
+                responseStatusCode <$> request
+
+            print response
```

Comment:
Can you add something more informative as a response? instead of just printing the number, print something like "Response Status: xxx"

It would also be nice to put a print statement before querying the service.. like "Request to xxxxx . . ."